images/server/install-packages.sh

```diff
@@ -43,8 +39,18 @@ fi
     "samba-client${samba_version_suffix}" \
     "samba-winbind${samba_version_suffix}" \
     "samba-winbind-clients${samba_version_suffix}" \
+    "samba-vfs-iouring${samba_version_suffix}" \
     tdb-tools \
-    "ctdb${samba_version_suffix}"
+    "ctdb${samba_version_suffix}")
+if [[ "${OS_BASE}" = centos ]]; then
+    dnf_cmd+=(--enablerepo=crb --enablerepo=resilientstorage)
+fi
```

Comment:
Purely a cosmetic change request but the above block can still stay after LINENO:32.

Reply:
done